packages/apps/patterns/react-composer/src/components/Markdown/codeMirrorDark.ts

```diff
@@ -0,0 +1,158 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { tags } from '@lezer/highlight';
+import get from 'lodash.get';
+
+import { tokens } from '../../styles';
+
+export const chalky = '#e5c07b';
```

Comment:
i realize this editor is rather unique -- and potentially user customizable, but i'm just curious how you think about families of colors.
should we have a package-global file to define colors?
example: i want at some point to adapt my "presenter" frame and may want to share a theme.

Reply:
Yeah, I was wondering about that myself… I think our Tailwind config should export its own palettes based on LAB space so these shade numbers actually mean something, then we'd also have these values available at runtime for situations like this.